File: perl/control_flow/for_loop/for_loop_formatting.pl
#
# Copyright (C) 2019-2026 Katarzyna Krasińska
# Languagium - https://github.com/katheroine/languagium
# Licensed under GPL-3.0 - see LICENSE.md
#

use strict;
use warnings;

my @r = 0..9;

for my $i (@r) {
  print("$i...\n");
}

print("\n");

for (@r) {
  print("$_...\n");
}

print("\n");

for (0..9) {
  print("$_...\n");
}

print("\n");

for (0..9) { print("$_...\n"); }

print("\n");

File: c#/control_flow/for_loop/ForLoopFormatting.cs
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

class ForLoopFormatting {
  public static void Main(string[] args) {
    int i;

    for (i = 0; i < 10; i++)
      System.Console.WriteLine(i + "...");

    System.Console.WriteLine();

    for (int j = 0; j < 10; j++)
      System.Console.WriteLine(j + "...");

    System.Console.WriteLine();

    for (int j = 0; j < 10; j++) System.Console.WriteLine(j + "...");

    System.Console.WriteLine();

    for (int j = 0; j < 10; j++)
    {
      System.Console.WriteLine(j + "...");
    }

    System.Console.WriteLine();
  }
}
